src/main/java/edu/iu/terracotta/service/app/impl/QuestionServiceImpl.java

```diff
@@ -97,12 +97,12 @@ public QuestionDto postQuestion(QuestionDto questionDto, long assessmentId, bool
     }
 
     @Override
-    public QuestionDto toDto(Question question, boolean answers, boolean student) {
-        return toDto(question, null, answers, student);
+    public QuestionDto toDto(Question question, boolean showCorrectAnswer, boolean student) {
```

Comment:
student is unused so it could be dropped

Reply:
Done!